slink-main/activation.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const sr = require('./smartrecruiters');
+const sapActivateEmployee = require('./sap/activateemployee');
+const util = require('./util');
+
+const process = async () => {
+  const applicants = await sr.getApplicantsOnboarding();
+  console.info(`Activation: Collected ${applicants.length} applicants from SmartRecruiters`);
+
+  const splitByFte = util.split(applicant => applicant.fullTime === true);
+  const ftes = splitByFte(applicants);
+
+  console.info(`Non-FTE applicants skipped: ${ftes.rejects.length}`);
+
+  const splitByEmployeeId = util.split(applicant => applicant.employeeId !== null);
+  const ftesWithId = splitByEmployeeId(applicants);
```

Comment:
Probably want `ftes.matches` here,  rather than `applicants`?

I argued with myself a long time about the implementation in `introduction.js`.  I ended up putting the split function creation first, then the use of the functions, so the flow of data is a _little_ more clear.  My JS foo is not super-strong.

I also struggled a long time with naming.

We can probably pull out the domain-specific split functions into a separate module and reuse between `introduction` and `activation`, as well.  The functions would use different "sides" of the split, at least WRT the employee ID check.